src/proxy.rs

```diff
@@ -120,13 +121,13 @@ async fn process_stream(
         .map(|addr| addr.to_string())
         .unwrap_or_else(|_| "".to_owned());
 
-    debug!("new connection from {}", source);
+    info!(source, "new connection from");
 
     let handshake = acceptor.accept(raw_stream).await;
 
     match handshake {
         Ok(inner_stream) => {
-            debug!("handshake success from: {}", source);
+            info!(source, "handshake success");
```

Comment:
do we wan't `info` here?

Reply:
Yes, all network connection try should be logged at info level to get more details